docs/pages/concepts/logical/credentials.md

```diff
@@ -7,4 +7,40 @@ abstract: Meshery uses one or more Credentials when authenticating to a managed
 language: en
 list: include
 ---
-Meshery uses one or more Credentials when authenticating to a managed or unmanaged Connection.
\ No newline at end of file
+Meshery uses one or more Credentials when authenticating to a managed or unmanaged Connection.
+Credentials are pivotal in Meshery as they serve as the authentication mechanism for connections registered with the Meshery server. They ensure secure interactions between Meshery and the various services and systems Meshery is designed to work with. <br>
```

Comment:
Extraneous, and perhaps, inaccurate to the extent that Meshery doesn't even use credentials sometimes.